101-load-balancer-cross-region/azuredeploy.json

```diff
@@ -0,0 +1,890 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "projectName": {
+            "type": "string",
+            "metadata": {
+                "description": "Specifies a project name that is used for generating resource names."
+            }
+        },
+        "location-cr": {
+            "type": "string",
+            "allowedValues": [
```

Comment:
why does this have allowedValues but the other locations do not?

Reply:
This one is for the cross region load balancer, which is only available in select regions.  The other two are normal load balancers that are available everywhere.

Reply:
Let's add some of that detail to the metadata description

Reply:
Edited the metadata.json and updated the description to include that the cross region load balancer is available in limited regions.  I also added that the regional load balancers can be in any region.